language_service/src/protocol.rs

```diff
@@ -2,15 +2,18 @@
 // Licensed under the MIT License.
 
 use qsc::line_column::Range;
-use qsc::{compile::Error, target::Profile, LanguageFeatures, PackageType};
+use qsc::PackageType;
+use qsc::{compile::Error, target::Profile, LanguageFeatures};
```

Comment:
nitpick: why did this one `use` get separated from the bunch?

```suggestion
use qsc::{compile::Error, target::Profile, LanguageFeatures, PackageType};
```

Reply:
This cheeky little import tried to make a run for it! Don't worry, I caught it and put it back in its braces.